src/main/java/pl/nextleveldev/smart_route/route/RouteService.java

```diff
@@ -0,0 +1,12 @@
+package pl.nextleveldev.smart_route.route;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class RouteService {
+    public List<Object> searchRoute(final String startLocation, final String destination) {
```

Comment:
Name suggest that You return only one route but in fact it is a List so the name should be "searchRoutes"